### DISCUSSION TITLE
is there a way to split the bar in half for notched screens

### DISCUSSION THREAD

**Discussion by Itstatertots** [FelixKratz/SketchyBar#582] [Q&A] [answered]
i want to have to have a bar on each side of the notch is there a way to split the bar in half

**Comment by maulik13** [comment]
Not directly. You can check the [tips for multiple bars](https://felixkratz.github.io/SketchyBar/config/tricks#multiple-bars), however considering that you cannot specify width for a bar it will not work. 

Technically, if you use positions "left", "q" and use brackets for those items you can simulate a bar to the left of the notch. And use "e" and "right" for rendering one on the right side. You just have to think creatively here :)

**Comment by FelixKratz** [comment, marked as answer]
Using brackets is the way to go, simply add this at the top of your `sketchybarrc` (before adding any other items):
```bash
sketchybar --add item left_anchor left --set left_anchor width=0 \
           --add item q_anchor q --set q_anchor width=0 \
           --add item e_anchor e --set e_anchor width=0 \
           --add item right_anchor right --set right_anchor width=0 \
           --add bracket left_bar left_anchor q_anchor \
           --set left_bar background.color=0xff000000 \
           --add bracket right_bar e_anchor right_anchor \
           --set right_bar background.color=0xffffffff
```
now you can control the properties of the `left_bar` and `right_bar` trough its bracket background properties.
